### ISSUE TITLE
Gettr remains broken

### ISSUE THREAD

**Issue by jocejocejoe** [WebBreacher/WhatsMyName#576]
First, mea culpa, I guess my new detection strings in the most recent commit 470ddc2 Fix Gettr detection are incorrect; I swear I saw "claimable" toggle true/false but now I don't. It was probably my mistake.

That said, something weird seems to be happening; the http return code is "-1" which looks like a program failure somewhere, but I am not sure how to find it. Of course "-1" is not a valid http return code. Any ideas? I'll re-fix Gettr (maybe just by reverting my last commit, sorry) if I can figure out why this is failing in this way.

```
# ./whats_my_name.py -a -u gettr -s gettr 
+-----------+------------------------------------------+----------+--------+
| Site Name | Url                                      | Category | Result |
+-----------+------------------------------------------+----------+--------+
| Gettr     | https://api.gettr.com/s/user/gettr/exist | social   | -1     |
+-----------+------------------------------------------+----------+--------+
```

**Comment by WebBreacher** [comment]
I believe (and @yooper can confirm this) that -1 means you didn't get a response back. Maybe they blocked you or blocked your VPN? Visiting https://api.gettr.com/s/user/gettr/exist directly from a browser, I see that he `claimable` param and valid should be correct. Also, using the whatsmyname.app website with the username `gettr` I see a positive result for the Gettr website so it is working in the web UI. When I run your command in WMN it works on the CLI too....so I think it is your system/network.

```
$ python3 whats_my_name.py -u gettr -s gettr
+-----------+------------------------------------------+----------+--------+
| Site Name | Url                                      | Category | Result |
+-----------+------------------------------------------+----------+--------+
| Gettr     | https://api.gettr.com/s/user/gettr/exist | social   | 200    |
+-----------+------------------------------------------+----------+--------+
```

**Comment by jocejocejoe** [comment]
wget and curl for the same URL work fine here, though. Even when adding the user agent whatsmyname uses. Very weird. Maybe it is something w/ my local python libraries.

Can you confirm that random usernames are properly **not** found on gettr, for you? I'm afraid I broke that.

**Comment by WebBreacher** [comment]
Yes you did break that as `"claimable": false` appears on both the correct and incorrect user pages. Please choose a different `e_string`

**Comment by jocejocejoe** [comment]
Ok, an older version of the OS is not exhibiting the "-1" problem, so I can test locally now. I'll get you an update soon. Thanks!

**Comment by jocejocejoe** [comment]
Should be fixed by pull request #577 . Sorry about that.
